platform/android/export/export_plugin.cpp

```diff
@@ -2373,6 +2383,19 @@ bool EditorExportPlatformAndroid::has_valid_project_configuration(const Ref<Edit
 		err += "\n";
 	}
 
+	String current_renderer = GLOBAL_GET("rendering/renderer/rendering_method.mobile");
+	if (current_renderer == "forward_plus") {
+		// Warning only, so don't override `valid`.
+		err += vformat(TTR("The \"%s\" renderer is designed for Desktop devices, and is not suitable for Android devices."), current_renderer);
```

Comment:
Should we check version here too? or first? If the user decides to try out the clustered renderer on Android, but then deploys to a pre SDK 24 phone, they may run into issues with that?

Reply:
Good point, I'll update the warning for the `forward_plus` to include the min SDK.